paddle/fluid/memory/allocation/allocator_facade.cc

```diff
@@ -160,6 +161,16 @@ class AllocatorFacadePrivate {
       platform::CUDAPlace p(i);
       system_allocators_[p] = std::make_shared<CUDAAllocator>(p);
     }
+#endif
+#if defined(PADDLE_WITH_ASCEND_CL)
+    system_allocators_[platform::NPUPinnedPlace()] =
+        std::make_shared<paddle::memory::allocation::NPUPinnedAllocator>();
+    int device_count = platform::GetNPUDeviceCount();
+    for (int i = 0; i < device_count; ++i) {
+      platform::NPUPlace p(i);
+      system_allocators_[p] =
+          std::make_shared<paddle::memory::allocation::NPUAllocator>(p);
+    }
```

Comment:
Why add pinned allocator here? Is it needed in batch_norm op?

Reply:
It is no longer needed and has been removed